ramls/item.json

```diff
@@ -176,6 +176,34 @@
             "type": "boolean",
             "description": "Records the fact that the note should only be displayed for staff",
             "default": false
+          },
+          "source": {
+            "type": "object",
+            "description": "The user who added the note",
+            "properties": {
+              "id": {
+                "type": "string",
+                "description": "The id of the user who added the note"
```

Comment:
Is this always the user who added the note, or does it change when the note is updated (to the user who last updated the note)?

Reply:
It changes when the note is updated as well. Updated the description.